### PR TITLE
gui/settings dialog; Add audio tab & small refactor.

### PR DESCRIPTION
**Pull request by Zangetsu38**
# About
- Add audio tab & small refactor on settings dialog.
- Remove double init of global volume value.

### CODE REVIEW

**Comment by nishinji** [comment]
In SDL case will not work until push save in settings.

File: vita3k/gui/src/settings_dialog.cpp [hunk 1]
```diff
@@ -185,6 +185,13 @@ static bool get_custom_config(GuiState &gui, EmuEnvState &emuenv, const std::str
                 config.fps_hack = gpu_child.attribute("fps-hack").as_bool();
             }
 
+            // Load Audio Config
+            if (!config_child.child("audio").empty()) {
+                const auto audio_child = config_child.child("audio");
+                config.audio_volume = audio_child.attribute("audio-volume").as_float();
```
Comment (bookmist) [inline vita3k/gui/src/settings_dialog.cpp:191]: You changed type of `audio_volume` class field to int, but here you load it as float

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp:191]: 😐🤦‍♂️😂
Sorry 😐🤦‍♂️

Comment (Zangetsu38) [reply vita3k/gui/src/settings_dialog.cpp]: Done 🗿

**Comment by Zangetsu38** [comment]
> In SDL case will not work until push save in settings.

Is normal that, code is set like that 🤔
I no touch this here

**Comment by nishinji** [comment]
> > In SDL case will not work until push save in settings.
> 
> Is normal that, code is set like that 🤔 I no touch this here

No, it works in master.
I don't know the details, but seemingly the volume value is not divided by 100 before press save.

**Comment by Zangetsu38** [comment]
> 
> No, it works in master. I don't know the details, but seemingly the volume value is not divided by 100 before press save.

I no underboobs what you said here, code is exactly same before
And click on save for apply change is totaly normal, i change nothing on this

**Comment by Zangetsu38** [comment]
> No, it works in master. I don't know the details, but seemingly the volume value is not divided by 100 before press save.

I no understund what you said here, code is exactly same before
And click on save for apply change is totaly normal, i change nothing on this

**Review by Macdu — APPROVED** [review]

**Pull request merged**